xcm/src/lib.rs

```diff
@@ -403,31 +327,39 @@ versioned_type! {
 	/// A single `MultiLocation` value, together with its version code.
 	#[derive(Ord, PartialOrd)]
 	pub enum VersionedMultiLocation {
+		#[codec(index = 1)] // v2 is same as v1 and therefore re-using the v1 index
 		V2(v2::MultiLocation),
+		#[codec(index = 3)]
 		V3(v3::MultiLocation),
 	}
 }
 
 versioned_type! {
 	/// A single `InteriorMultiLocation` value, together with its version code.
 	pub enum VersionedInteriorMultiLocation {
+		#[codec(index = 2)] // while this is same as v1::Junctions, VersionedInteriorMultiLocation is introduced in v3
```

Comment:
this could be 1. let me know

Reply:
Was there a `VersionedJunction` in XCMv2? If so, then it should be 1, otherwise this is ok.

Reply:
<img width="303" alt="Screenshot 2023-03-16 at 12 16 25 AM" src="https://user-images.githubusercontent.com/1201310/225293480-65c4914e-1b48-45e7-92fd-864f34d171d4.png">

all the versioned type in v0.9.37